### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by bdrtc**

### CODE REVIEW

File: draft-murillo-whep.md [hunk 1]
```diff
@@ -229,15 +229,13 @@ a=msid:- d46fb922-d52a-4e9c-aa87-444eadc1521b
 ~~~~~
 {: title="HTTP POST and PATCH doing SDP O/A example"}
 
-## SDP offer generated by the WHEP endpint
+## SDP offer generated by the WHEP endpoint
 
-If the WHEP player preffers the WHEP Endpoint to generate the SDP offer, the WHEP player will send a POST request without HTTP BODY and an Accept HTTP header of "application/sdp" to the configured WHEP endpoint URL.
+If the WHEP player prefers the WHEP Endpoint to generate the SDP offer, the WHEP player will send a POST request without HTTP BODY and an Accept HTTP header of "application/sdp" to the configured WHEP endpoint URL. The HTTP POST request MUST have a content type of "application/sdp".
```
Comment (murillo128) [inline draft-murillo-whep.md:234]: If the WHEP player is acting as a offerer, the HTTP POST will be empty, and there would be no content type header. That is why the Accept http header is set instead

File: draft-murillo-whep.md [hunk 2]
```diff
@@ -229,15 +229,13 @@ a=msid:- d46fb922-d52a-4e9c-aa87-444eadc1521b
...
+If the WHEP player prefers the WHEP Endpoint to generate the SDP offer, the WHEP player will send a POST request without HTTP BODY and an Accept HTTP header of "application/sdp" to the configured WHEP endpoint URL. The HTTP POST request MUST have a content type of "application/sdp". 
 
-The HTTP POST request MUST have a content type of "application/sdp" and contain the SDP offer as the body. 
+The WHEP endpoint will generate an SDP offer according to the JSEP rules and return a "201 Created" response with a content type of "application/sdp", the SDP offer as the body, a Location header field pointing to the newly created resource and an Expire header indicating the maximum time that the WHEP player is allowed to send the SDP answer to the WHEP resource.
 
-The WHEP endpoint will generate an SDP offer according to the JSEP rules and return a "201 Created" response with a content type of "application/sdp", the SDP offer as the body, a Location header field pointing to the newly created resource and a Expire header indicating the maximum time that the WHEP player is allowed to send the SDP answer to the WHIP resource.
```
Comment (murillo128) [inline draft-murillo-whep.md:238]: Why is this removed?

File: draft-murillo-whep.md [hunk 3]
```diff
@@ -489,16 +487,14 @@ NOTE: The naming of both the "rel" attribute value of "ice-server" and the targe
...
+It might be also possible to configure the STUN/TURN server URLs with long-term credentials provided by either the broadcasting service or an external TURN provider on the WHEP player, overriding the values provided by the WHEP endpoint.
 
 ## Authentication and authorization
 
 WHEP endpoints and resources MAY require the HTTP request to be authenticated using an HTTP Authorization header field with a Bearer token as specified in {{!RFC6750}} section 2.1. WHEP players MUST implement this authentication and authorization mechanism and send the HTTP Authorization header field in all HTTP requests sent to either the WHEP endpoint or resource except the preflight OPTIONS requests for CORS.
 
-WHEP endpoints and resources MAY require the HTTP request to be authenticated using an HTTP Authorization header field with a Bearer token as specified in {{!RFC6750}} section 2.1. WHEP players MUST implement this authentication and authorization mechanism and send the HTTP Authorization header field in all HTTP requests sent to either the WHEP endpoint or resource.
```
Comment (murillo128) [inline draft-murillo-whep.md:500]: Why is this removed?

**Review by murillo128 — CHANGES_REQUESTED** [review]

File: draft-murillo-whep.md (hunk 3 above)
Comment (bdrtc) [reply draft-murillo-whep.md:500]: It's duplicated with the above line.

**Comment by murillo128** [comment]
Thank you for the PR! I have commit it and reverted the change of the Content-Type header when the HTTP post contains no body.

File: draft-murillo-whep.md (hunk 3 above)
Comment (murillo128) [reply draft-murillo-whep.md:500]: Yes, I realized about this after sending my review comment. Thank you for the review!

File: draft-murillo-whep.md (hunk 2 above)
Comment (bdrtc) [reply draft-murillo-whep.md:238]: It's above, not removed.

File: draft-murillo-whep.md (hunk 1 above)
Comment (bdrtc) [reply draft-murillo-whep.md:234]: Okay, actually it's from the next line. I'll delete it.
